assets/i18n/en.json

```diff
@@ -474,6 +474,7 @@
     "I18N_SIGNUP_USERNAME_EXPLANATION": "Your username will be shown next to your contributions.",
     "I18N_SIGNUP_WAIVER_OBJECTIVE": "The waiver of the attribution (BY) requirement means that, if someone reuses this work, they are not required to attribute the authors. However, all of your individual contributions to explorations will be available on this site in the exploration change log, and people who reuse the exploration are encouraged (but not required) to include a link to this page.",
     "I18N_SIGNUP_WHY_LICENSE": "Why CC-BY-SA?",
+    "I18N_SOLICIT_ANSWER_DETAILS_QUESTION": "Hey, how did you landed on this answer?",
```

Comment:
how did you land..
I had a previous comment as well mentioning this change.

Reply:
Changed